packages/office-ui-fabric-react/src/components/MessageBar/MessageBar.scss

```diff
@@ -139,7 +139,7 @@
     @include focus-border();
     min-width: 0;
   }
-  .ms-MessageBar-actionsOneline .ms-MessageBar-dismissal {
+  .ms-MessageBar-dismissalOneline .ms-MessageBar-dismissal {
```

Comment:
Was this supposed to be a state or modifier? Probably out of scope for this PR, just doesn't look right.